docs/migration-guide-1.24-2.0.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Migration Guide 1.24-2.0
+---
+
+Once you upgrade to Kyma 2.0, perform the manual steps described in the Migration Guide.
+
+## Security
+
+### Native Kubernetes authentication in Kyma
+
+Kyma 2.0 simplifies things and switches to the native Kubernetes authentication. As a result, we no longer support the following authentication and authorization components:
+
+- API Server Proxy
+- Console Backend Service
+- Dex
+- IAM Kubeconfig Service
+- Permission Controller
+
+To use the native Kubernetes authentication in Kyma, you need to remove the deprecated components manually.
+
+After the successful upgrade to Kyma 2.0, run the following [script](assets/1.24-2.0-remove-deprecated-resources.sh), which uninstalls and deletes the unsupported items.
+
+### ORY Oathkeeper without Dex
```

Comment:
Add warning about evaluation profile and Oauth2Client - ory maester restart can be required.

Reply:
Moved to the end of the section